content/en/blog/kernel-testing.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Kernel testing framework
+date: 2023-09-14
+author: Federico Di Pierro, Aldo Lacuku
+slug: kernel-testing-framework
```

Comment:
can you add tags like the recent posts

Reply:
How can i add it?
And where can i find supported tags? :D

Reply:
@Issif ?

Reply:
Here: https://github.com/falcosecurity/falco-website/blob/master/data/en/blog_tags.yaml